### PR TITLE
HoC 2024 - Create hourofcode.com/resources and deprecate /promote

### PR DESCRIPTION
**Pull request by kelbyhawn**
Creates https://hourofcode.com/resources for the 2024 Hour of Code. Also updates the "Resources" link in the header nav to point to this new page.

## Links
Jira ticket: [ACQ-2311](https://codedotorg.atlassian.net/browse/ACQ-2311)
Figma mock: [here](https://www.figma.com/design/emvQNdidNnPvoU1NucF7Re/Hour-of-Code-2024?node-id=2710-2186&t=TEZOkDq1TVC0Oai1-1)

## Testing story
Tested locally to make sure this page works responsively and with LTR and RTL languages.

## Followup
I'll deprecate and redirect the existing /promote and /promote/resources pages in a followup ticket: [ACQ-2414](https://codedotorg.atlassian.net/browse/ACQ-2414)

----

localhost hourofcode com_3000_us_en_resources

### Note: I think there's a bug with Mac OS's record feature where it's not showing the cursor pointers on hover, this is working when not recording.

https://github.com/user-attachments/assets/5321056f-08fd-4914-9c8b-06ed6d10bdde

https://github.com/user-attachments/assets/7ea69591-1fcb-43ae-924d-4e7409d6ef68

https://github.com/user-attachments/assets/604df4ca-02a1-4810-8c2f-f4694e12d8b9

### CODE REVIEW

File: pegasus/sites.v3/hourofcode.com/public/css/page/how-to.scss [hunk 1]
```diff
@@ -79,6 +79,10 @@ nav.choose-guide {
       background: var(--neutral_white);
       border-color: var(--neutral_white);
     }
+
+    &:is(:hover, :active):not(.active) {
+      background: rgba(255, 255, 255, 0.4);
+    }
```
Comment (kelbyhawn) [inline pegasus/sites.v3/hourofcode.com/public/css/page/how-to.scss:85]: Sneaking these How-to guide hover state styles into this PR: 

https://github.com/user-attachments/assets/224ff0ee-5a01-401d-abf1-02ddebb6f435

File: pegasus/sites.v3/hourofcode.com/public/promote/index.md.partial [hunk 2]
```diff
@@ -6,9 +6,9 @@ nav: promote_nav
 
 {{ signup_button }}
 
-# Get your community involved in the Hour of Code  
+# Get your community involved in the Hour of Code
```
Comment (bethanyaconnor) [inline pegasus/sites.v3/hourofcode.com/public/promote/index.md.partial:9]: Is this page translated at all?

Comment (bethanyaconnor) [reply pegasus/sites.v3/hourofcode.com/public/promote/index.md.partial:9]: I'm also now realizing all the changes in this PR are whitespace

**Review by bethanyaconnor — APPROVED** [review]
🎉

File: pegasus/sites.v3/hourofcode.com/public/promote/index.md.partial (hunk 2 above)
Comment (kelbyhawn) [reply pegasus/sites.v3/hourofcode.com/public/promote/index.md.partial:9]: I'm going to delete and redirect this page and all of it's translated files in this ticket: [ACQ-2414](https://codedotorg.atlassian.net/browse/ACQ-2414)

**Pull request merged**